framework/src/base/MooseVariableInterface.C

```diff
@@ -155,6 +155,16 @@ MooseVariableInterface::secondTest()
   return const_cast<VariableTestSecond &>(_variable->secondPhi());
 }
 
+VariableTestSecond &
+MooseVariableInterface::secondTestFace()
+{
+  if (_nodal)
+    mooseError("Nodal variables do not have second derivatives");
+
+  return const_cast<VariableTestSecond &>(_variable->secondPhiFace());
```

Comment:
I just assumed we were doing this for some reason.  I think I'd rather update them all in a separate PR anyway -- it's going to change a lot of user code when we do...